dts/bindings/gpio/zephyr,gpio-emul.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020, Friedt Professional Engineering Services, Inc
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO Emulator
+
+compatible: "zephyr,gpio-emul"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+    reg:
+      required: true
+
+    label:
+      required: true
+
+    rising-edge:
```

Comment:
Sorry, I missed this.  Please provide at least basic documentation on the properties, e.g.:

    description: Enables support for rising edge interrupt detection